README.md

```diff
@@ -1,3 +1,42 @@
 # Articulate Golang Images
 
 Base golang image which contains common dependencies used in our organization.
+
+## Using Vault
+
+In order to use vault to pull in secrets as ENV variables you will need to include the following Atlas variables in your EB settings.
+
+- The ENV variables `VAULT_ADDR` and `VAULT_TOKEN` will need to be defined in atlas variables and in the terraform module as Elasticbeanstalk environment settings. Here is an example of the settings for the terraform module. These will be used by consul-template to connect to vault.
+ ```
+setting {
+  namespace = "aws:elasticbeanstalk:application:environment"
+  name = "APP_NAME"
+  value = "<your app name>"
+  }
+setting {
+  namespace = "aws:elasticbeanstalk:application:environment"
+  name = "VAULT_TOKEN"
+  value = "${lookup(var.my_app_vault_token, var.env)}"
+  }
+setting {
+  namespace = "aws:elasticbeanstalk:application:environment"
+  name = "VAULT_ADDR"
+  value = "${lookup(var.vault_addr, var.env)}"
+  }
+```
+The app name value should be equal to the repo name which should also be equal to the app name in the vault path used to store the variables.
```

Comment:
It might clarify to state this as "The app name, repo name and name in the vault path should all be the same."